File: help/pt_pt/exercicios_da_ficha_4/Ficha04_005/Ficha04_005.js
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    document.write("Número \t: " );// print text
    num = parseInt(prompt("Número \t: " ,"0") );
    
    document.write(num + "\n");// print input 
    if(num == 0 ){
        document.write("Nulo" );
    }else{
        if(num > 0 ){
            document.write("Positivo" );
        }else{
            document.write("Negativo" );
        }
    }
}

//JavaScript  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Mon, 2022 Sep 26 15:31

File: help/pt_pt/exercicios_da_ficha_4/Ficha04_005/Ficha04_005.cs
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

using System.IO;
using System;
public class Ficha04_005 {

    //Main Function
    static void Main() {
        Console.Write("Número \t: " );
        int num = int.Parse(Console.ReadLine());
        if(num == 0 ){
            Console.Write("Nulo" );
        }else{
            if(num > 0 ){
                Console.Write("Positivo" );
            }else{
                Console.Write("Negativo" );
            }
        }
    }

}
//C#  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Mon, 2022 Sep 26 15:31
